### ISSUE TITLE
bad format in "missing name in package.json" error

### ISSUE THREAD

**Issue by lukeed** [oven-sh/bun#6081]
### What version of Bun is running?

1.0.2+37edd5a6e389265738e89265bcbdf2999cb81a49

### What platform is your computer?

Darwin 21.3.0 arm64 arm

### What steps can reproduce the bug?

```jsonc
// services/crawler/package.json
{
	"private": true,
	"type": "module"
}

// package.json
{
  "workspaces": ["services/*"]
}
```

```sh
$ bun install
```

**Result**

```
error: Missing "name" from package.json in services//crawler
{
^
```

### What is the expected behavior?

Format to `serivices/name`, not `services//name`

**Comment by axlEscalada** [comment]
workaround:  add the `name` field in `services/crawler/package.json` and it works

**Comment by lukeed** [comment]
yes of course but thats not what the issue is for.

**Comment by dylan-conway** [comment]
fixed in `v1.1.10` 
<img width="533" alt="Screenshot 2024-05-24 at 9 40 54 PM" src="https://github.com/oven-sh/bun/assets/35280289/fbf7b9d7-7e8b-494a-84c5-8cad2a5e2bd7">
